commands/rm.go

```diff
@@ -16,10 +16,18 @@ func cmdRm(c CommandLine, api libmachine.API) error {
 	force := c.Bool("force")
 	confirm := c.Bool("y")
 
+	if force {
+		confirm = true
+	}
+
 	for _, hostName := range c.Args() {
-		h, err := api.Load(hostName)
-		if err != nil {
-			return fmt.Errorf("Error removing host %q: %s", hostName, err)
+		h, loaderr := api.Load(hostName)
+		if loaderr != nil {
+			// On --force, continue to remove on-disk files/dir
+			if !force {
+				return fmt.Errorf("Error removing host %q: %s", hostName, loaderr)
+			}
+			log.Errorf("Error removing host %q: %s. Continuing on `-f`, host instance may by running", hostName, loaderr)
 		}
 
 		if !confirm && !force {
```

Comment:
Since this checks for both `confirm` and `force`, why is it needed to set `confirm` on line 20 above?